docs/docs/live-examples/opentelemetry-store/use-cases/add-item-into-shopping-cart.md

```diff
@@ -0,0 +1,92 @@
+# OpenTelemetry Store - Add item into the shopping cart
+
+In this use case, we want to validate the following story:
+
+```
+As a consumer
+I want to choose a product from the catalog
+And add it to my shopping cart
+So I can continue to explore the catalog to fulfill my shopping list
+```
+
+You can trigger this use case by calling the endpoint `POST /api/cart` from the Frontend service, with the following request body:
+```json
+{
+  "item": {
+    "productId": "OLJCESPC7Z",
+    "quantity": 1
+  },
+  "userId": "2491f868-88f1-4345-8836-d5d8511a9f83"
+}
+```
+
+It should return a payload similar to this:
+```json
+{
+  "userId": "2491f868-88f1-4345-8836-d5d8511a9f83",
+  "items": [
+    {
+      "productId": "OLJCESPC7Z",
+      "quantity": 1
+    }
+  ]
+}
+```
+
+## Building a Test for This Scenario
+
+Using Tracetest, we can [create a test](../../../web-ui/creating-tests.md) that will execute an API call on `POST /api/cart` and validate the following properties:
+- The correct ProductID was sent to the Product Catalog API.
+- The product persisted correctly on the shopping cart.
+
+### Traces
+
+Running these tests for the first time will create an Observability trace like the image above, where you can see spans for the API calls (HTTP and gRPC), and database calls:
```

Comment:
is `image above` correct? or `image below`?

Reply:
good catch, it is below, thanks!